Add Gallery page (#85)
kasenvr/vircadia-docs-sphinx: docs/source/explore.rst

> Vircadia is a metaverse where you can connect and create with others. We invite you to explore VR worlds created in Vircadia and interact with other users. You can visit your friend's VR world, meet people, attend events and even go for a class on avatar creation. It's an immersive and interactive experience with realistic visuals and audio.

Reviewer: ```suggestion
Vircadia is a metaverse where you can connect and create with others. We invite you to explore the worlds created in Vircadia and interact with other users. You can visit friends' virtual worlds, meet people, attend events, and learn about creation in virtuality. It's an immersive and interactive experience with realistic visuals and audio.
```

> If its your first time using Vircadia, you'll start your journey a local space by yourself, where you can experiment with the controls and learn about the product. When you are ready, try exploring other domains using the Explore app and socialize with people around the metaverse. If you've visited Vircadia before, you will return to the location where you last visited.

Reviewer: ```suggestion
If it's your first time using Vircadia, you'll start your journey in a local tutorial space by yourself, where you can experiment with the controls and learn about the Interface. When you are ready, try exploring other domains using the Explore app and socialize with people around the metaverse. If you've visited Vircadia before, you will return to the location where you last visited.
```
